43/README.md

```diff
@@ -29,15 +30,12 @@ The ZRE protocol provides a way for a set of nodes on a local network to discove
 * To be neutral with respect to operating system, programming language, and hardware.
 * To allow any number of nodes to run in one process, to allow large-scale simulation and testing.
 
-## Changes Over Version 1
+## Changes Over Version 2
 
-This is version 2 of ZRE. The changes over version 1 are:
+This is version 3 of ZRE. The changes over version 2 are:
 
-* The 'strings' and 'dictionary' types redesigned for better expandability.
-* The version number was increased to 2.
-* The hard-coded dependency on TCP transport has been removed (merging ipaddress and port into a single endpoint field).
-* Removed the logging extension, which is better handled by a separate framework.
-* Protocol signature is 1 rather than 0.
+* The version number was increased to 3.
+* Becon packets support advertising of a 32-octet CURVE public key for the purposes of establishing CURVE enabled security to the connection.
```

Comment:
now that you say that- i do remember seeing parts of that in the code. *face-palm*